### DISCUSSION TITLE
Add support for 'outer margin' for block with floating notifications

### DISCUSSION THREAD

**Discussion by Seme4eg** [ErikReider/SwayNotificationCenter#252] [Q&A] [answered]
Currently i found a way to only setup margin for every separate floating notification, but is there a way to make the 'block' with floating notifications to have top margin? There is a way using `.notification-row:first-child {margin-top: 60px;}`, but it also applies to first notification in control-center, which i don't want.

**Comment by ErikReider** [comment]
The CSS `floating-notifications` class should do the trick :)

**Comment by Seme4eg** [comment]
![image](https://user-images.githubusercontent.com/24554850/236407330-0b0d3db9-740f-4b1f-bb50-b58278d66996.png)
am i missing something?

**Comment by Seme4eg** [comment]
![image](https://user-images.githubusercontent.com/24554850/236409680-75ea0164-5c7e-4a23-afe9-91e813dbcebf.png)

**Comment by ErikReider** [comment, marked as answer]
You're right, for some reason the `margin-top` isn't respected. Try this:
```css
.floating-notifications > scrolledwindow > viewport > box {
  margin-top: 150px;
}
```

For future reference, running swaync like this `GTK_DEBUG=interactive swaync` will open a inspector window that'll allow you to see all of the CSS classes + other information :)

**Comment by Seme4eg** [comment]
did work indeed, and thanks for the tip!

and i think that trick with gtk_debug should be in readme so people who actually read readme can help themselfes even better)
